src/thunderbots/software/util/canvas_messenger/canvas_messenger.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "thunderbots_msgs/CanvasLayer.h"
+
+// Forward declaration
+namespace ros
+{
+    class NodeHandle;
+    class Publisher;
+}  // namespace ros
+
+namespace Util
+{
+    using CanvasLayer = thunderbots_msgs::CanvasLayer;
+
+    class CanvasMessenger
+    {
+       public:
+        typedef struct Sprite
```

Comment:
optional, but you can remove the `Sprite` here in C++:
``` suggestion
        typedef struct
```

Reply:
Didn't seem to compile when I did that...

Reply:
odd........... it's fine anyhow